packages/kit/test/apps/options/test/test.js

```diff
@@ -9,9 +9,15 @@ test.describe('base path', () => {
 	test('serves a useful 404 when visiting unprefixed path', async ({ request }) => {
 		const html = await request.get('/slash/', { headers: { Accept: 'text/html' } });
 		expect(html.status()).toBe(404);
-		expect(await html.text()).toBe(
-			'Not found (did you mean <a href="/path-base/slash/">/path-base/slash/</a>?)'
-		);
+		if (process.env.DEV) {
+			// Vite's message
+			expect(await html.text()).toBe(
+				'The server is configured with a public base URL of /path-base - did you mean to visit <a href=\"/path-base/slash/\">/path-base/slash/</a> instead?'
+			);
+		} else {
+			// SvelteKit's message
+			expect(await html.text()).toBe('Not found (did you mean <a href=\"/path-base/slash/\">/path-base/slash/</a>?)');
```

Comment:
i think if we're going to do this we should just use the same message as Vite

Reply:
done